src/OmnichannelChatSDK.ts

```diff
@@ -1621,16 +1624,20 @@ class OmnichannelChatSDK {
         let conversationId;
 
         try {
-            const {LiveWSAndLiveChatEngJoin: liveWSAndLiveChatEngJoin} = this.liveChatConfig;
-            const {msdyn_postconversationsurveyenable, msfp_sourcesurveyidentifier, postConversationSurveyOwnerId} = liveWSAndLiveChatEngJoin;
+            const chatConfig: ChatConfig = this.liveChatConfig;
+            const {LiveWSAndLiveChatEngJoin: liveWSAndLiveChatEngJoin, ChatWidgetLanguage: chatWidgetLanguage} = chatConfig;
+            const {msdyn_postconversationsurveyenable, msfp_sourcesurveyidentifier, msfp_botsourcesurveyidentifier, postConversationSurveyOwnerId, postConversationBotSurveyOwnerId} = liveWSAndLiveChatEngJoin;
+            const {msdyn_localeid} = chatWidgetLanguage;
+            const localeId: string = msdyn_localeid ?? "1033";
```

Comment:
Can we use `defaultLocaleId` from `locale.ts`? It should be defaulted to `1033`.